pkg/services/validator/oci.go

```diff
@@ -287,7 +287,7 @@ func readOciRegistryRule(c *components.OCIPluginConfig, r *plug.OciRegistryRule,
 		return err
 	}
 	if authSecretName != notApplicable {
-		r.Auth = plug.Auth{SecretName: authSecretName}
+		r.Auth = plug.Auth{SecretName: &authSecretName}
```

Comment:
why was this updated to a pointer?

Reply:
Because you made that field a pointer earlier today!

Reply:
https://github.com/validator-labs/validator-plugin-oci/pull/250/files#diff-b9eacffdc159581b219713d78795c3044883a3bb70960606b4c517c05b24a225R97